### PR TITLE
Updates to Readme

### PR DESCRIPTION
**Pull request by hotzenklotz**
Changes:
- updated README
- updated `pyproject.toml`to include readme on PyPi page
- added a script run all "extended" cuber testing scripts easily on your local machine
- tweaked GitHub CI, removed CircleCI builds
- update some dev dependencies:
  - pytest 
  - black (reformatted 4 files)  
  - pylint (fixed 4 files)

### CODE REVIEW

**Comment by hotzenklotz** [comment]
@normanrz Please review. I will open a separate PR for more API documentation, but let's get this merged first. Also waiting for @rschwanhold and PR #249 .

File: README.md [hunk 1]
```diff
@@ -32,7 +32,9 @@ Created with [Python3](https://www.python.org/).
 * NIFTI files
 
 ## Installation
-### Python3 with pip
+### Python 3 with pip from PyPi
+- `wkcuber`requires at least Python 3.6+
```
Comment (normanrz) [inline README.md:36]:
```suggestion
- `wkcuber` requires at least Python 3.6+
```

File: wkcuber/downsampling.py [hunk 2]
```diff
@@ -541,10 +541,11 @@ def downsample_mags(
                 scale = read_datasource_properties(path)["scale"]
             except Exception as exc:
                 logging.error(
-                    f"Could not get the scale from the datasource-properties.json. Probably your path is wrong. "
+                    "Could not get the scale from the datasource-properties.json. Probably your path is wrong. "
```
Comment (normanrz) [inline wkcuber/downsampling.py:544]: No more f-string?

Comment (hotzenklotz) [reply wkcuber/downsampling.py:544]: Pylint complains!

**Review by normanrz — APPROVED** [review]

**Pull request merged**